cni/network/invoker_cns.go

```diff
@@ -51,14 +51,15 @@ func NewCNSInvoker(podName, namespace string) (*CNSIPAMInvoker, error) {
 //Add uses the requestipconfig API in cns, and returns ipv4 and a nil ipv6 as CNS doesn't support IPv6 yet
 func (invoker *CNSIPAMInvoker) Add(nwCfg *cni.NetworkConfig, args *cniSkel.CmdArgs, hostSubnetPrefix *net.IPNet, options map[string]interface{}) (*cniTypesCurr.Result, *cniTypesCurr.Result, error) {
 
+	endpointId := GetEndpointID(args)
+
 	// Parse Pod arguments.
-	podInfo := cns.KubernetesPodInfo{PodName: invoker.podName, PodNamespace: invoker.podNamespace}
+	podInfo := cns.NewPodInfo(args.ContainerID, endpointId, invoker.podName, invoker.podNamespace)
 	orchestratorContext, err := json.Marshal(podInfo)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	endpointId := GetEndpointID(args)
 	ipconfig := cns.IPConfigRequest{
 		OrchestratorContext: orchestratorContext,
```

Comment:
Similarly here, OrchestratorContext info is changed here. It is meant to be podName + PodNS